src/config/store/devtools.ts

```diff
@@ -0,0 +1,9 @@
+import { StoreDevtoolsModule } from '@ngrx/store-devtools';
+import { environment } from '../../environments/environment';
+
+export const StoreDevModules = [
+  StoreDevtoolsModule.instrument({
+    maxAge: 1000,
+    logOnly: environment.production,
```

Comment:
Won't this always be false?  DevTools aren't enabled in production, so `environment.production` should always return false.   Should we make this an explicit `false` just to make that clearer?